src/foam/dashboard/view/DAOTable.js

```diff
@@ -200,7 +200,7 @@ foam.CLASS({
       code: function() {
         var self = this;
         if ( ! this.dao ) return;
-        this.dao.where(this.predicate).limit(this.limit).select().then((objects) => {
+        this.dao.limit(this.limit).select().then((objects) => {
```

Comment:
Doesnt this remove predicate support from DAOTable? I would leave this here and just override it in GroupedDAOTable

Reply:
right, thank you. I will do that.